chart/values.yaml

```diff
@@ -246,14 +246,17 @@ components:
   workspace:
     ports:
       http:
-        containerPort: 23000
         supervisorPort: 22999
+        containerPort: 23000
+        staticFrontendPort: 24000
```

Comment:
Is the static frontend the reduced frontend app, i.e. not the IDE? In that case it should be served on the current 23000. the new port (24000) should be named IDEPort or similar.

Reply:
we can swap them and rename things after it works somehow :) currently just trying to make a progress as easy as possible